src/mpid/ch4/netmod/ofi/ofi_recv.h

```diff
@@ -30,7 +30,7 @@
 #define FUNCNAME MPIDI_OFI_recv_iov
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
-MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_recv_iov(void *buf, MPI_Aint count,
+MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_recv_iov(void *buf, MPI_Aint count, size_t data_sz,
```

Comment:
Just curious, would it be possible to remove the `count` argument to this function after this addition and just work in bytes?

Reply:
I think this `count` is useful for multiple other purposes. But as you mentioned in today's ch4 call, we can create another patch to reduce arguments if needed.